pyclibrary/c_parser.py

```diff
@@ -589,33 +318,20 @@ def load_cache(self, cache_file, check_validity=False):
                     return False
 
             # Import all parse results
-            self.import_dict(cache['file_defs'])
+            self.clib_intf.include(cache['clib_intf'])
             return True
 
         except Exception:
             logger.exception("Warning--cache read failed:")
             return False
 
-    def import_dict(self, data):
-        """Import definitions from a dictionary.
-
-        The dict format should be the same as CParser.file_defs.
-        Used internally; does not need to be called manually.
-
-        """
-        for f in data.keys():
-            self.current_file = f
-            for k in self.data_list:
-                for n in data[f][k]:
-                    self.add_def(k, n, data[f][k][n])
-
     def write_cache(self, cache_file):
```

Comment:
I thought we don't need caching at all in future.

In your use case, where you have header file information, that shall not be delivered with your software, you could solve the caching requirement as commented in the code sample of https://github.com/MatthieuDartiailh/pyclibrary/issues/12#issuecomment-111876243

Or do you know a use case, where this is not matching our requirements?